src/main/java/org/kpmp/repositoryDataset/RepositoryDatasetService.java

```diff
@@ -56,9 +62,19 @@ public RepositoryDatasetService(
 	}
 
   public List<RepositoryDataset> getRepositoryDataset() throws Exception {
-    List <RepositoryDataset> datasets = new ArrayList<>();
+    List <RepositoryDataset> datasets = new ArrayList<>(); 
```

Comment:
This line and the next could be replaced with a simple
``List <RepositoryDataset> datasets = fileRepo.findAll();``